api_app/views.py

```diff
@@ -82,7 +83,19 @@ def _multi_analysis_request(
     else:
         # fire celery task
         for index, job in enumerate(jobs):
-            runtime_configuration = runtime_configurations[index]
+            runtime_configuration = {}
+
+            for analyzer in job.analyzers_to_execute:
+                # Appending custom config to runtime configuration
+                config = get_custom_config_as_dict(
+                    user, CustomConfig.Type.ANALYZER, name=analyzer
+                ).get(analyzer, {})
+                if analyzer in runtime_configurations[index]:
+                    config = config | runtime_configurations[index][analyzer]
+                if config:
+                    runtime_configuration[analyzer] = config
```

Comment:
this is complex logic that requires proper testing. So please extract this part to a separate function and do the tests of what you want that this would behave.